### PR TITLE
Settings page: Added Contributors

### LINKED ISSUE #191: Settings Page: Add Contributors
After issue #190 is completed using GitHub contributors API add all the contributors below the admins.
with their GitHub ids. 

if they want their LinkedIn to be added then wish to drop them to here.

### PR DESCRIPTION
**Pull request by Udit-singh**
### Fixes #191 

Added contributors list view with github contributors API,
@anushbhatia hope you review it and merge it.

### CODE REVIEW

**Comment by arteevraina** [comment]
Hey @Udit-singh! Welcome to the repo. Can you provide with any screen shot of your change ?

File: lib/settings/contact_developers.dart [hunk 1]
```diff
@@ -1,9 +1,37 @@
...
+    const contributorsAPIUrl =
+        'https://api.github.com/repos/smaranjitghose/DocLense/contributors';
+    final response = await http.get(contributorsAPIUrl);
+
+    if (response.statusCode == 200) {
+      setState(() {
+        jsonContributors = json.decode(response.body) as List;
+      });
+    } else {
+      throw Exception('Failed to load Contributors from API');
```
Comment (arteevraina) [inline lib/settings/contact_developers.dart]: How about showing a toast message when there is an exception?

**Review by arteevraina — CHANGES_REQUESTED** [review]

**Comment by Udit-singh** [comment]
@arteevraina I have Incorporated following requested changes and attached a snapshot below.

https://user-images.githubusercontent.com/55452299/113503824-f463c180-9551-11eb-8345-d0a5613f4010.mp4

**Review by arteevraina — APPROVED** [review]

**Comment by anushbhatia** [comment]
Remove our names if possible. Smaranjit and Anush.

**Comment by Udit-singh** [comment]
> Remove our names if possible. Smaranjit and Anush.

@anushbhatia I have Incorporated following requested changes.

**Pull request merged**